modules/prepare_contamination.nf

```diff
@@ -78,9 +78,6 @@ process check_own {
 process concat_contamination {
   label 'minimap2'
   
-  publishDir "${params.output}/", mode: params.publish_dir_mode, pattern: "db.fa.gz"
-  publishDir "${params.output}/", mode: params.publish_dir_mode, pattern: "db.fa.fai"
```

Comment:
agree... it's nice to have directly the files available to use them in a genome viewer

Reply:
Agreed, I'll publish these someplace reasonable and skip them if intermediate files are turned off

Reply:
For now I publish them here:
```
results
├── clean
│   ├── ERR10008447.fasta.gz
│   └── ERR10035961.fasta.gz
├── intermediate
│   ├── host.fa.fai
│   ├── host.fa.gz
```
I changed the filenames to `host.fa.*` because I think `db.fa.*` is not very descriptive.